### PR TITLE
Synchronize MTI stats with webrtc-stats cleanup

### PR DESCRIPTION
**Pull request by dontcallmedom**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/pull/2744.html" title="Last updated on Jun 16, 2022, 6:49 AM UTC (d3f3bc0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2744/c508d44...d3f3bc0.html" title="Last updated on Jun 16, 2022, 6:49 AM UTC (d3f3bc0)">Diff</a>

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -15505,7 +15503,6 @@ <h3>
                 {{RTCReceivedRtpStreamStats/packetsReceived}},
                 {{RTCReceivedRtpStreamStats/packetsLost}},
                 {{RTCReceivedRtpStreamStats/jitter}},
-                {{RTCReceivedRtpStreamStats/packetsDiscarded}},
```
Comment (jan-ivar) [inline webrtc.html:15508]: `packetsDiscarded` seems [well implemented](https://wpt.fyi/results/webrtc/RTCPeerConnection-mandatory-getStats.https.html?label=master&label=experimental&aligned).

**Review by jan-ivar — CHANGES_REQUESTED** [review]

File: webrtc.html (hunk 1 above)
Comment (jan-ivar) [reply webrtc.html:15508]: It's also [still in the stats spec](https://w3c.github.io/webrtc-stats/#dom-rtcinboundrtpstreamstats-packetsdiscarded) AFAICT

Comment (henbos) [reply webrtc.html:15508]: It's implemented in inbound-rtp, not remote-inbound-rtp. The WPT tests both, one passes on all browsers and one fails on all browsers.

The PR moved it from RTCReceivedRtpStreamStats which implied both inbound-rtp and remote-inbound-rtp to only RTCInboundRtpStreamStats. This matches the implementation status

**Review by henbos — APPROVED** [review]
AFAICT this is correct. Approved!

File: webrtc.html (hunk 1 above)
Comment (jan-ivar) [reply webrtc.html:15508]: Thanks for explaining! LGTM then

File: webrtc.html [hunk 2]
```diff
@@ -15620,23 +15617,6 @@ <h3>
...
-            <tr>
-              <th class="row">
-                {{RTCStatsType/"sender"}}
-              </th>
-              <td rowspan="2">
-                {{RTCMediaHandlerStats}}
-              </td>
-              <td rowspan="2" data-tests=
-              "RTCPeerConnection-track-stats.https.html,RTCPeerConnection-mandatory-getStats.https.html">
-                {{RTCMediaHandlerStats/trackIdentifier}}
```
Comment (henbos) [inline webrtc.html:15632]: Oh we should make it mandatory to implement RTCInboundRtpStreamStats.trackIdentifier. It got moved there.

**Review by henbos — CHANGES_REQUESTED** [review]
I noticed one thing we should change IMO

File: webrtc.html (hunk 2 above)
Comment (dontcallmedom) [reply webrtc.html:15632]: can you provide it as a separate PR since that one got merged now? :)

Comment (henbos) [reply webrtc.html:15632]: Will do https://github.com/w3c/webrtc-pc/issues/2747

Comment (henbos) [reply webrtc.html:15632]: Done: https://github.com/w3c/webrtc-pc/pull/2748

**Pull request merged**